Queueomatic/Client/Pages/Room.razor

```diff
@@ -0,0 +1,72 @@
+﻿@*there needs to be a check if the room owner is connecting when participantname is omitted*@
+@page "/room/{RoomId}"
+@page "/room/{RoomId}/{ParticipantName}"
```

Comment:
Same as above. 
Why should be a path for a participant?

Reply:
How else would you supply the page with a participant name? We decided that you enter your name on the landing page and that name needs to be used on this page.

Reply:
Yeah, you're right but we get all participants of the room from the room itself.

Reply:
Yes we do, but that's done on the room page

Reply:
Why shouldn't we add the participant when we ask for thier name, already on landing page?

Reply:
You could, but you would need to retrieve the created participant from the room page. How would you know which one to retrieve? 

Either you create the participant on the landing page and supply the path with a participant id or you supply the name in the route and create it in the room. 

Either way it is the same process, but I find it better to isolate as much code as possible to its respective page.